storybook/stories/Layout/InputWithLabel.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { InputWithLabel } from "../../../src/components/Layout/InputWithLabel";
+import { size } from "../../../src/common/styles";
+import { View } from "react-native";
+
+const inputWithLabelElements: JSX.Element[] = [
+  <InputWithLabel key="0" label="Input with label" />,
+  <InputWithLabel key="1" label="Not Editable Input" editable={false} />,
+];
+
+storiesOf("Layout", module).add("InputWithLabel", () => (
+  <View style={{ margin: size(3) }}>
```

Comment:
this just a wrapper view

Reply:
verticalMargin to only for top and bottom.. we need margin in all side

Reply:
as in, instead of just `margin: size(3)`, can change to `marginHorizontal: size(3), marginVertical: size(4)`. I notice that the spacing felt a bit off.